"""
    Contains all custom exception types within this package
"""

from .bad_args import BadArgsError
